bin/create.js

```diff
@@ -8,284 +8,220 @@ const { ensureDir, readdir } = require("fs-extra");
+
+        // Ensure the output directory exists
+        await ensureDir(outputDir);
+        await isOutputDirectoryEmpty(outputDir);
+
+        // Showing the loader
+        const spinner = loading(
+            `Generating custom Celo Composer project with the following packages: ${selectedPackages.join(
+                ", "
+            )}...\n`
+        );
+
+        // Shell commands to clone and trim the required directories
+        shell.cd(pwd);
+        shell.exec(
+            `git clone --depth 2 --filter=blob:none --sparse ${BASE_URL} ${projectName}`
+        );
+        shell.cd(projectName);
+
+        let packageJson = {
+            name: `${projectName}`,
+            version: "1.0.0",
+            description: "Custom Celo Composer project.",
+            private: true,
+            author: "Celo",
+            license: "MIT",
+            scripts: {},
+            repository: {
+                type: "git",
+                url: "git+https://github.com/celo-org/celo-composer.git",
+            },
+            bugs: {
+                url: "https://github.com/celo-org/celo-composer/issues",
+            },
+            homepage:
+                "https://github.com/celo-org/celo-composer/blob/main/README.md",
+            workspaces: {
+                packages: ["packages/*"],
+            },
+            keywords: ["celo-composer", "celo"],
+        };
+
+        for (let x = 0; x < selectedPackages.length; x++) {
+            // clone to local only the projects user wants
+            const { stdout, stderr, code } = shell.exec(
+                `git sparse-checkout add packages/${selectedPackages[x]}`,
+                { silent: true }
+            );
+
+            // flutter project doesn't have package.json
+            if (selectedPackages[x] != availablePackages["flutter-app"]) {
+                // update name of each package.json project to work properly with monorepo
+                let packageFile = shell.cat(
+                    `packages/${selectedPackages[x]}/package.json`
+                );
+                let projectPackage = JSON.parse(packageFile.stdout);
+                projectPackage.name = `@${projectName}/${selectedPackages[x]}`;
+            }
+
+            // if project isn't web no need to netlify.toml
+            if (
+                selectedPackages[x] == availablePackages["react-native-app"] ||
+                selectedPackages[x] ==
+                    availablePackages["react-native-app-without-expo"] ||
+                selectedPackages[x] == availablePackages["flutter-app"]
+            ) {
+                shell.rm("netlify.toml");
+            }
+
+            /**
+             * Getting all packages selected by the user
+             * First list them via echo packages/\*\/
+             * Some string manipulation so that packages looks like
+             * eg:- ["react-app", "hardhat"] etc...
+             */
+
+            let packages = shell
```

Comment:
@nestorbonilla here is the package.json script generation code.

Reply:
Reviewed.